google/cloud/storage/emulator/README.md

```diff
@@ -84,3 +84,40 @@ For N==1 and N==2 behave like `return-305-after-256K`, for `N>=3` ignore the
 failure instruction and return successfully. This is used to test failures during
 retry, the client cooperates by sending the retry counter in the failure
 instructions.
+
+
+## Retry Test API
+
+### Creating a new Retry Test
+
+```bash
+    curl -X POST "http://localhost:9000/retry_test" -H 'Content-Type: application/json' \
```

Comment:
I think block quoting via indentation and via backticks is overkill, maybe?

Reply:
Yea... looks like i just couldn't decide which to use. I'll address this issue.